### DISCUSSION TITLE
Download of actions on “Set up” job is failing. Throwing 503's

### DISCUSSION THREAD

**Discussion by szohaib** [community/community#25744] [Actions]
<p>Seems like there is a problem with Github Actions? But Github Status pages show everything is normal. Is anyone facing the same issue? Can someone please help me out here?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/2/e282956acc692d35699b585185935207547207b3.png" data-download-href="/uploads/short-url/wjNGjCIBnyYY354v3DkBokWzVCj.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/17105550/181092991-24cdca76-40b8-4fd5-8f5f-9f52a613462a.png" alt="image" data-base62-sha1="wjNGjCIBnyYY354v3DkBokWzVCj" width="690" height="270" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e282956acc692d35699b585185935207547207b3_2_690x270.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e282956acc692d35699b585185935207547207b3_2_1035x405.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e282956acc692d35699b585185935207547207b3_2_1380x540.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e282956acc692d35699b585185935207547207b3_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1510×591 30 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by kf6kjg** [comment]
<p>I think this is a global problem on GitHub ATM. Even though their status page isn’t showing it.</p>

**Comment by nativvstudios** [comment]
<p><em>phew</em> I didn’t screw up my repo lol. Same here<br>
edit: finally showing up on the status page<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/7/679f2255df5186c3f4e473e58e2ab879c8a5b442.png" data-download-href="/uploads/short-url/eMG7BOQmDREfw78ciyZLeXbGvMC.png?dl=1" title="brave_O2AUq3SYts" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/26895364/181092994-03f1b68d-460a-40e2-92f8-bd887d31dabb.png" alt="brave_O2AUq3SYts" data-base62-sha1="eMG7BOQmDREfw78ciyZLeXbGvMC" width="690" height="286" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/7/679f2255df5186c3f4e473e58e2ab879c8a5b442_2_690x286.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/7/679f2255df5186c3f4e473e58e2ab879c8a5b442_2_1035x429.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/7/679f2255df5186c3f4e473e58e2ab879c8a5b442_2_1380x572.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/7/679f2255df5186c3f4e473e58e2ab879c8a5b442_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">brave_O2AUq3SYts</span><span class="informations">1464×608 33.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by szohaib** [comment]
<p>Lol, that’s kinda comforting! I thought I messed up my repo</p>

**Comment by kf6kjg** [comment]
<p>All the actions URLs resolve to <a href="https://codeload.github.com/" rel="noopener nofollow ugc">https://codeload.github.com/</a> which is throwing angry unicorns at everyone.</p>

**Comment by jeromemiranda** [comment]
<p>Having the same issue, worse is we need to put something to production today.</p>

**Comment by BlackSwan2019** [comment]
<p>My night shouldn’t have ended like this. :smiley:</p>

**Comment by szohaib** [comment]
<p>Early morning here! Bad start to the day :slightly_frowning_face:</p>

**Comment by BlackSwan2019** [comment, marked as answer]
<p>Jobs are working again; able to checkout repos.</p>
